tests/datamodules/test_levircd.py

```diff
@@ -64,14 +64,10 @@ def test_val_dataloader(self, datamodule: LEVIRCDPlusDataModule) -> None:
         batch = next(iter(datamodule.val_dataloader()))
         batch = datamodule.on_after_batch_transfer(batch, 0)
         if datamodule.val_split_pct > 0.0:
-            assert (
-                batch['image1'].shape[-2:] == batch['mask'].shape[-2:] == (1024, 1024)
-            )
-            assert batch['image1'].shape[0] == batch['mask'].shape[0] == 1
-            assert (
-                batch['image2'].shape[-2:] == batch['mask'].shape[-2:] == (1024, 1024)
-            )
-            assert batch['image2'].shape[0] == batch['mask'].shape[0] == 1
+            assert batch['image1'].shape[-2:] == batch['mask'].shape[-2:] == (256, 256)
```

Comment:
Previously val/test was being done on the full 1024x1024 images. `ExtractPatches` fixes this so now the val/test images are patches instead.